wiki/Achievements/en.md

```diff
@@ -109,12 +114,11 @@ Mod Introduction
 | ![Hidden](/wiki/Achievements/img/all-intro-hidden.png "Hidden") | | |
 | ![Sudden Death](/wiki/Achievements/img/all-intro-suddendeath.png "Sudden Death") ![Perfect](/wiki/Achievements/img/all-intro-perfect.png "Perfect") | | |
 
-
 Hush-Hush
-----------
-
-Achievements of this kind have no obvious way to get them. The player himself must find out what he needs to do in order to unlock the achievement. The title itself is a clue.
+---------
 
+Achievements of this kind have no obvious way to get them. The player himself must find out what he needs to do in order to unlock the achievement.
```

Comment:
`Achievements of this kind have no obvious way to get them. The player must find out what needs to be done in order to unlock the achievement.`